docs/usage/python.md

```diff
@@ -74,12 +82,13 @@ e.g.:
 
 <!-- prettier-ignore -->
 !!! tip
-    The index-url found in the `requirements.txt` file takes precedence over a `registryUrl` configured like the above.
-    To override the URL found in `requirements.txt`, you need to configure it in `packageRules`, as they are applied _after_ package file extraction.
+    If a `requirements.txt` file has a index-url then Renovate follows that link, instead of following any link set in the `registryUrls` array.
+    To override the URL found in `requirements.txt`, you must create a custom `packageRules` setting.
```

Comment:
I think this tip should get an example as well. After reading this text you still don't know _what_ you should put in the `packageRules`.

Reply:
add a link to private packages page and add samples there

Reply:
Which page do you mean, can you give me a link?

And can you give me the example to put in the private packages page?

Reply:
https://docs.renovatebot.com/getting-started/private-packages/